app/jobs/dradis/plugins/csv/mapping_import_job.rb

```diff
@@ -0,0 +1,91 @@
+module Dradis::Plugins::CSV
+  class MappingImportJob < ApplicationJob
+    queue_as :dradis_project
+
+    # mappings hash:
+    # The key is the column index, with a hash containing the type of resource (evidence/issue/node)
+    # its supposed to map to and the dradis field for the resource (only for evidence and issues).
+    #
+    # e.g.
+    # {
+    # '0' => { 'type' => 'node' },
+    # '1' => { 'type' => 'issue', field: 'Title' },
+    # '2' => { 'type' => 'evidence', field: 'Port' }
```

Comment:
```suggestion
    # '1' => { 'type' => 'issue', 'field' => 'Title' },
    # '2' => { 'type' => 'evidence', 'field' => 'Port' }
```
or is the `field` key needed to be a symbol?

Reply:
It's string only, will update the comment.